### DISCUSSION TITLE
Multipart

### DISCUSSION THREAD

**Discussion by Piioo** [savonrb/savon#995] [Q&A]
The `[savon-multipart](https://github.com/savonrb/savon-multipart)` gem was working find until savon version 2.12.1. In version 2.13.x was added a new feature https://github.com/savonrb/savon/pull/761 which do not work together with this multipart gem.
https://github.com/savonrb/savon-multipart/issues/29

Is there a possible way in savon to achieve what this gem was doing without this gem?

**Comment by pcai** [comment]
There might be, if you contribute it yourself. Or at the least it would be helpful if you opened a pr to add a failing test case for someone else
